### DISCUSSION TITLE
v4 with NextJS v15.0.3 | ERROR: Package subpath './lib/util/toPath' is not defined by "exports

### DISCUSSION THREAD

**Discussion by therealgilles** [tailwindlabs/tailwindcss#15232] [Help]
I'm trying out v4 beta and getting the following error with NextJs v15.0.3 in dev:

```
[28.11.2024 19:16.38.222] [ERROR]  ⨯ ./src/middleware.ts
Error: ./src/middleware.ts: Package subpath './lib/util/toPath' is not defined by "exports" in ./node_modules/tailwindcss/package.json
    at transformFile.next (<anonymous>)
    at run.next (<anonymous>)
```

Any clue or advice would be very much appreciated.

**Comment by samyosm** [comment]
The automatic migration gives me the same error :/

**Comment by bicarlsen** [comment]
For me, this occurred because a plugin I was using ([tailwind-scrollbar](https://github.com/adoxography/tailwind-scrollbar/issues/104)) was incompatible with `4.0.0`.

To discover this I commented out all the `@plugin` directives in my main tailwind file, then re-enabled them one-by-one.
